src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -158,6 +159,21 @@ public class DurableTaskExtension :
         /// </value>
         public bool DisableHttpManagementApis { get; set; }
 
+        /// <summary>
+        /// Gets or sets a value of Event Grid Topic Endpoint for emitting lifecycle events.
+        /// If this property has been set, it will emit the event to the Event Grid Topic.
+        /// </summary>
+        public string EventGridTopicEndpoint { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value of Event Grid Key.
+        /// If this property has been set, it will emit the event to the Event Grid Topic.
+        /// </summary>
+        public string EventGridKeySettingName { get; set; }
+
+        internal LifeCycleNotificationHelper LifeCycleTraceHelper => this.lifeCycleTraceHelper;

```

Comment:
This property should be renamed to match the new class name: `LifeCycleNotificationHelper`.

Reply:
Thank you! We will change it.